### ISSUE TITLE
Make motd configurable

### ISSUE THREAD

**Issue by martinbean** [laravel/homestead#1082]
I’d like to make my Homestead’s `motd` configurable, rather than use the one with the giant “HOMESTEAD” banner, hosted on a [non-Laravel domain](https://homestead.joeferguson.me).

I install Homestead on a [per-project basis](https://laravel.com/docs/master/homestead#per-project-installation), so would be good to add project-specific MOTDs to those VMs.

**Comment by svpernova09** [comment]
Happy to take suggestions on the motd, but the primary goal of the customized motd is to alert Homestead users to issues that may be affecting them so currently I would rather not override this.

You can override the motd via `after.sh`

**Comment by martinbean** [comment]
If Laravel’s all about developer happiness, then it should be a little more easily configured than adding commands in an **after.sh** file.

Maybe if there’s a **.motd** file or similar in the root of the project, Homestead uses that in place of the Homestead MOTD when creating the VM?

**Comment by deviantintegral** [comment]
Note that the motd itself is built at `/etc/update-motd.d` so you can drop a file in there with project-specific info without having to remove the existing one, or you could remove `50-motd-news` as well.

**Comment by svpernova09** [comment]
> If Laravel’s all about developer happiness, then it should be a little more easily configured than adding commands in an after.sh file.

This doesn't mean *your* happiness is more important than *mine*.

**Comment by martinbean** [comment]
Well, gee. Isn’t that just a welcoming comment 🙄

**Comment by svpernova09** [comment]
> Well, gee. Isn’t that just a welcoming comment 🙄

You might find the environment more welcoming if you toned down your entitlement. I do this *for free* in my own spare time. I don't expect everyone to agree with me, but you will be respectful of the people who build the tools you're using or you're going to find yourself pretty alone in the open source world.

I'm happy to hear feature suggestions. Hell I have to practically pull them out of people sometimes. But when you get upset that I don't share your view shows your immaturity. You asked about a new feature and the maintainer responded within ~30 minutes with an explanation of why it wouldn't make sense and you immediately start attacking and trying to troll me. Do you think you were getting me in trouble?

Entitlement is a dangerous attitude to have in open source. No one owes you anything. I'm here providing services for free because I feel like I owe a debt to the OSS maintainers who came before me and I want to make sure OSS is still here for the next generation. You can take your crap attitude and go elsewhere, it's not appreciated or wanted here. However if you'd like to go back to civil discussion about Homestead features I'm happy to, and will be waiting for your next feature suggestion or school yard insult, whichever is more important to you is what I expect to see first.

**Comment by martinbean** [comment]
@svpernova09 I merely opened an issue to suggest making something configurable. I haven’t “attacked” you or “trolled” you or thrown “school yard insults”. I’m sorry if you feel as though I have, but I just opened an issue to ask for something to be made configurable. If I’ve worded it so that it came across as entitled then I apologise, as that genuinely wasn’t my intention.

That being said, I don’t appreciate you speaking down to me in a condescending manner. I never attacked you or your commitment to open source software, so don’t appreciate being spoken to as though I’m a child making their first foray into the software world. I’ve over ten years’ professional experience, and pretty much all of that time working with—and contributing to—open source software. It may not be as much experience as yourself, but this isn’t exactly my first rodeo either.

If you’re going to accuse me of an entitled attribute then dismissive comments like, “This doesn't mean _your_ happiness is more important than _mine_” doesn’t really paint _you_ in a good light. _That_ felt like an attack and shut-down, which is why I replied in an admittedly flippant manner. But don’t accuse me of “attacking” and “trolling” you when I’ve done _nothing_ of the sort.
